packages/core/src/js/integrations/expocontext.ts

```diff
@@ -85,19 +88,19 @@ function getExpoUpdatesContext(): ExpoUpdatesContext {
     is_using_embedded_assets: !!expoUpdates.isUsingEmbeddedAssets,
   };
 
-  if (typeof expoUpdates.updateId === 'string') {
-    updatesContext.update_id = expoUpdates.updateId;
+  if (typeof expoUpdates.updateId === 'string' && expoUpdates.updateId) {
+    updatesContext.update_id = expoUpdates.updateId.toLocaleLowerCase();
```

Comment:
q: Why not just toLowerCase? It should be a GUID only and be language-independent.
```suggestion
    updatesContext.update_id = expoUpdates.updateId.toLowerCase();
```

Reply:
Thank you, yes that the local was not intentional.